packages/extension-ui/src/components/HelperFooter.tsx

```diff
@@ -9,29 +9,30 @@ import styled from 'styled-components';
 interface Props extends ThemeProps {
   className?: string;
   children: React.ReactNode;
+  tooltipOnly?: boolean;
 }
 
-function HelperFooter({ children, className }: Props): React.ReactElement<Props> {
+function HelperFooter({ children, className, tooltipOnly }: Props): React.ReactElement<Props> {
   return <div className={className}>{children}</div>;
 }
 
 export default styled(HelperFooter)(
-  ({ theme }: Props) => `
+  ({ theme, tooltipOnly }: Props) => `
```

Comment:
Could you explain how this relates to any kind of a tooltip? I was unable to infer that from the code, but maybe I missed something 🧐

Reply:
![image](https://user-images.githubusercontent.com/50427204/217842000-037d693b-2123-44ca-a5ea-19f51fa65a0c.png)
it is referred as a tooltip on the designs so I went with that.
I'm open to suggestions tho 👯

Reply:
Oh, I think it just clicked for me - _tooltip only_ was meant to indicate that there's nothing under that component and no margin is necessary to push the content down?

I'd then suggest a different, more scalable approach, however requiring more work/code so up to you whether to employ it :) Namely, any properties influencing _external layout / position_ should be imposed by the consumers of the component in question - after all, the component itself doesn't know where or how it's laid out in the application. E.g. ``const Consumer = styled(TheComponent)`margin-bottom: 16px;'` `` where needed.

Reply:
Just explaining a why I believe that ☝️ is a good practice: This enables you to lay out the component in virtually any environment, without need o changing its internals and thus potentially adapting all other usages and/or growing it's interface with usage-specific cases. You want it spaced with a margin - you got it; you want it spaced with a gap in a grid - you got it; you want it not spaced at all - still got it; none of those cases requiring changes in the component, keeping it truly reusable.

Reply:
Sounds great! I'll try to adapt to that 😄